CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## 0.13.0 (November 16, 2022)
+
+### Enhancements
+ * Add new snippet for `@QuarkusIntegrationTest`. See [#754](https://github.com/redhat-developer/quarkus-ls/issues/754).
+ * Generic support for Java data model in Qute templates. See [#503](https://github.com/redhat-developer/quarkus-ls/issues/503).
+ * Hover operator for `#case`, `#is`. See [#716](https://github.com/redhat-developer/quarkus-ls/issues/716).
+ * Clickable inlay hint for Java type in Qute templates. See [#702](https://github.com/redhat-developer/quarkus-ls/issues/702).
+ * Validation, completion & definition support for enum in #switch section. See [#689](https://github.com/redhat-developer/quarkus-ls/issues/689), [#690](https://github.com/redhat-developer/quarkus-ls/issues/690), [#691](https://github.com/redhat-developer/quarkus-ls/issues/691).
+ * Create a code action class per error code. See [#688](https://github.com/redhat-developer/quarkus-ls/pull/688).
+ * Base "generate missing member" code actions on whether the type is a source or binary file. See [#676](https://github.com/redhat-developer/quarkus-ls/issues/676).

```

Comment:
This can be grouped in with "Code actions to create Java field / getter method / template extension"